lms/djangoapps/discussion_api/api.py

```diff
@@ -9,6 +9,7 @@
 from django.core.urlresolvers import reverse
 from django.http import Http404
 import itertools
+from lms.djangoapps.django_comment_client.base.views import track_voted_event
```

Comment:
There are imports for the track functions on line 29. Can you merge this with those?